release_dashboard/lib/services/dev_local_conductor.dart

```diff
@@ -31,10 +31,20 @@ class DevLocalConductorService extends LocalConductorService {
     );
 
     final FrameworkRepository localFrameworkUpstream =
-        FrameworkRepository(checkouts, localUpstream: true, additionalRequiredLocalBranches: [candidateBranch]);
+        FrameworkRepository(
+          checkouts,
+          localUpstream: true,
+          name: 'framework-upstream',
+          additionalRequiredLocalBranches: [candidateBranch],
+        );
 
     final EngineRepository localEngineUpstream =
-        EngineRepository(checkouts, localUpstream: true, additionalRequiredLocalBranches: [candidateBranch]);
+        EngineRepository(
+          checkouts,
+          localUpstream: true,
+          name: 'engine-upstream',
+          additionalRequiredLocalBranches: [candidateBranch],
+        );
 
     // TODO: turn force to false
```

Comment:
This should follow the flutter todo style (TODO(yugue): explaination, issue url)

Reply:
Done, that was a temporary placeholder.

Reply:
Yeah, I told Alex that I add these invalid TODO comments to WIP PRs so that I know I have to fix them before merging.